tasks/config/template-config.yml

```
@@ -49,8 +49,9 @@
     state: directory
     owner: "{{ nginx_config_main_template.user | default('nginx') }}"
     mode: 0755
-  loop:
-    "{{ nginx_config_http_template }}"
+  loop: "{{ nginx_config_http_template | map(attribute='config') }}"
```

The extra attribute you added breaks the task logic
```suggestion
  loop: "{{ nginx_config_http_template }}"
```

---

Yes, you are right.